signature/internal/sequoia/gosequoia.c

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
`signature/internal/sequoia/*.[ch]` are auto-generated in https://github.com/ueno/podman-sequoia/tree/main/go/sequoia , only copied into this repo.